components/Collections/src/MDCCollectionViewFlowLayout.m

```diff
@@ -394,7 +394,7 @@ - (MDCCollectionViewLayoutAttributes *)updateSupplementaryViewAttribute:
   // view rows.
   CGRect insetFrame = attr.frame;
   if (!CGRectIsEmpty(insetFrame)) {
-    UIEdgeInsets insets = self.sectionInset;
```

Comment:
I think the original code is better here—seeing an uninitialized local variable makes me read the rest of the code just to figure out if it got assigned in all cases. Plus future edits to the following logic might change things and miss that the variable wasn't assigned.